src/clj/game/core/drawing.clj

```diff
@@ -70,14 +70,14 @@
          (if suppress-event
            (effect-completed state side eid)
            (let [draw-event (if (= side :corp) :corp-draw :runner-draw)]
-             (swap! state assoc-in [side :register :currently-drawing] drawn)
+             (swap! state update-in [side :register :currently-drawing] conj drawn)
```

Comment:
I don't think this is going to give us the list of lists that we want. Calling `conj` on two list will merge the lists together into one. When we first populate `:currently-drawing` we need to wrap the list in another list. I think that's why the test is failing in the latest build.

I guess it needs to look something like this
```
(conj [[1 2] [3 4]] [5 6])       
;;=> [[1 2] [3 4] [5 6]]
```

Reply:
If `:currently-drawing` is `nil`, `conj` will wrap the given object in a list. If it's a list, `conj` will append the object.